cowprotocol/accounting/slippage/slippage_prices_4064601.sql

```diff
@@ -135,5 +187,5 @@ native_token_prices as (
 )
 
 select * from prices
-union all
+union distinct
```

Comment:
Why was this change required? Before, the entries in the prices table were always distinct from entries in the native_token_prices table since the token address was different. Does this still hold?

If not, there might be additional issues due to a token having multiple prices at the same time, leading to duplicate entries when joining.

Reply:
Tbh i didn't think too much about it and it just looked more correct to me (in the sense that if you provide a prices table where native tokens are included, then you should not use the corresponding native_token_prices entries). In the current state, i don't think it can ever happen that the two table have common entries so i will switch back to union all